qemu/tests/boot_with_disable_ept.py

```diff
@@ -14,11 +14,12 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment
     """
+    timeout = float(params.get("login_timeout", 2400))
```

Comment:
params.get_numeric("login_timeout", 2400, float), fyi.
Either is OK for me

Reply:
Thanks xueqiang highlight it, record it.